Fab/Fab.jsx

```diff
@@ -11,13 +12,23 @@ class Fab extends MaterialComponent {
     super();
     this.componentName = "fab";
     this._mdcProps = ["mini", "exited"];
+    this.themeProps = ["primary", "secondary"];
   }
   componentDidMount() {
     super.attachRipple();
   }
   materialDom(props) {
+    let className = props.class || props.className || "";
```

Comment:
We should omit `class` and `className` from `props`. See https://github.com/prateekbh/preact-material-components/blob/master/Select/Select.jsx#L112 for how to omit the properties (please note that I clone the props as to be certain that I don't modify the actual props). Otherwise `class` is not being set at all (I tested this).

Reply:
This works if you pass `className` to Fab instead of `class`  🤔

Reply:
not sure why, it should treat `class` as a normal prop however

Reply:
Yea its being stripped out of `props` for some reason

Reply:
I found the culprit https://github.com/prateekbh/preact-material-components/blob/master/MaterialComponent.js#L64

Reply:
oh heck! yeah i did that to make sure we do not override class accidentally just like this.

Reply:
So this is desired behavior then? Intended use is to just pass `className` prop?

Reply:
yeah just use `className`. I'll make sure that this gets merged properly

Reply:
It still doesn't use the `class` prop as it gets deleted by the above mentioned code. If we remove `delete componentProps.class;` it will add the user-specified classes correctly.

Reply:
However, I am not certain what side effects that might have on the other components.